internal/e2e/e2e.go

```diff
@@ -0,0 +1,62 @@
+// Package e2e provides a few utilities for use in unit tests.
+package e2e
+
+import (
+	"path/filepath"
+	"runtime"
+
+	"github.com/supabase/auth/internal/conf"
+	"github.com/supabase/auth/internal/storage"
+	"github.com/supabase/auth/internal/storage/test"
+)
+
+var (
+	projectRoot string
+	configPath  string
+)
+
+func init() {
+	_, thisFile, _, _ := runtime.Caller(0)
```

Comment:
Can we somehow make sure this init doesn't accidentally run in the main executable?

Reply:
I've added a runtime check, let me know if this doesn't go far enough and we can consider build tags.